lib/sections/home.dart

```diff
@@ -1,15 +1,333 @@
+                border: Border.all(
+              color: Colors.black,
+            )),
+            child: Align(
+                alignment: Alignment.center,
+                child: Row(
+                  children: [
+                    Container(
+                      child:
+                          Image.asset(_image, fit: BoxFit.fitHeight, width: 90),
+                      padding: const EdgeInsets.symmetric(
+                          horizontal: 30.0, vertical: 0.0),
+                    ),
+                    Text(_country,
+                        textScaleFactor: 1.8, textAlign: TextAlign.center)
+                  ],
+                ))));
+  }
+}
+
+// FIRST LAYER
+// List of categories (eg. population)
+class CategoryPage extends StatelessWidget {
+  // No state (like list of countries), return list of tags
+  Widget build(BuildContext context) {
+    return Container(
+        child: Center(
+            child: ListView(
+      children: [CategoryTag("Population"), CategoryTag("Mortality")],
+    )));
+  }
+}
+
+// Create tag listing a category
+// TODO: Convert to stateless if possible
+class CategoryTag extends StatefulWidget {
+  String _category = "";
+
+  CategoryTag(String category) {
+    this._category = category;
+  }
+
+  @override
+  _CategoryTagState createState() => _CategoryTagState(this._category);
+}
+
+// State for category tag
+class _CategoryTagState extends State<CategoryTag> {
+  String _category = "";
+
+  _CategoryTagState(String category) {
+    this._category = category;
+  }
+
+  // Function to push section for category
+  void _pushInfo() {
+    Navigator.of(context)
+        .push(MaterialPageRoute<void>(builder: (BuildContext context) {
+      return Scaffold(
+        appBar: AppBar(
+          title: Text(_category),
+          centerTitle: true,
+        ),
+        body: Center(
+            // Container with info on category
+            child: CategoryInfoPage()),
+        // Navbar
+        bottomNavigationBar: BottomNavigationBar(
+          unselectedItemColor: Colors.black,
+          selectedItemColor: Colors.green,
+          items: const <BottomNavigationBarItem>[
+            BottomNavigationBarItem(
+              icon: Icon(Icons.home),
+              label: 'Home',
+            ),
+            BottomNavigationBarItem(
+              icon: Icon(Icons.query_stats),
+              label: 'Compare',
+            ),
+            BottomNavigationBarItem(
+              icon: Icon(Icons.folder_open),
+              label: 'Reporting',
+            ),
+            BottomNavigationBarItem(
+              icon: Icon(Icons.more),
+              label: 'More',
+            ),
+          ],
+          // TODO: fix navbar if possible
+          // currentIndex: _selectedIndex,
+          // onTap: _onItemTapped,
+        ),
+      );
+    }));
+  }
+
+  // Tag with name of category, detects tap
+  Widget build(BuildContext context) {
+    return GestureDetector(
```

Comment:
Any way to separate GestureDetector() out into its own function so that you don't have to copy-paste for CountryTag and CategoryTag?